packages/config/constants/TripTypes.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-export type TripTypes = 'Return' | 'OneWay' | 'MultiCity';
+export type TripType = 'Return' | 'OneWay' | 'MultiCity';
 export const TRIP_TYPES = {
```

Comment:
When you are on it. We can probably rename this to `TRIP_TYPE` to have it unified. But it's just an idea and definitely not necessary for this PR 😉 .